### PR TITLE
Clean and upgrade dependencies

### PR DESCRIPTION
**Pull request by leleuj**

### CODE REVIEW

File: pac4j-cas/pom.xml [hunk 1]
```diff
@@ -27,6 +27,10 @@
     <packaging>jar</packaging>
     <name>pac4j for CAS protocol</name>
 
+    <properties>
+        <cas.version>3.4.1</cas.version>
```
Comment (mmoayyed) [inline pac4j-cas/pom.xml]: You might want to bump this to 3.6 at least.

File: pac4j-openid/pom.xml [hunk 2]
```diff
@@ -35,18 +40,27 @@
...
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
-            <groupId>javax.servlet</groupId>
-            <artifactId>javax.servlet-api</artifactId>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
```
Comment (mmoayyed) [inline pac4j-openid/pom.xml]: This should be marked as provided.

File: pac4j-oauth/pom.xml [hunk 3]
```diff
@@ -27,15 +27,16 @@
 	<packaging>jar</packaging>
 	<name>pac4j for OAuth protocol</name>
 
+	<properties>
+		<scribe.version>2.2.2</scribe.version>
+		<jackson.version>2.7.1-1</jackson.version>
```
Comment (mmoayyed) [inline pac4j-oauth/pom.xml]: You likely should stay with 2.6 on jackson. 2.7 is not supported by Spring yet, and I imagine that might break a lot of existing uses.

File: pac4j-cas/pom.xml (hunk 1 above)
Comment (leleuj) [inline pac4j-cas/pom.xml]: I'm referring to the CAS client: the latest version is the 3.4.1, isn't it?

File: pac4j-openid/pom.xml (hunk 2 above)
Comment (leleuj) [inline pac4j-openid/pom.xml]: If I exclude the `commons-logging` dependency for `openid4java`, the `jcl-over-slf4j` is mandatory, isn't it? So I shouldn't mark it as `provided`, should I?

File: pac4j-oauth/pom.xml (hunk 3 above)
Comment (leleuj) [inline pac4j-oauth/pom.xml]: Yes, you're right. I will change the dependency version.

File: pac4j-cas/pom.xml (hunk 1 above)
Comment (mmoayyed) [inline pac4j-cas/pom.xml]: True. Ignore this plz. 
--
Sent with Air Mail

**Comment by leleuj** [comment]
@mmoayyed : are we good on this one?

**Comment by mmoayyed** [comment]
Yup. Please proceed.

**Pull request merged**